browser/components/MetricsLists.tsx

```diff
@@ -14,6 +14,8 @@
 import * as React from 'react';
 import SimpleLineChart from '../components/SimpleLineChart';
 
+import RadzChart from '../components/RadzChart';
```

Comment:
Lets use a more descriptive name for this component.

Reply:
what do you think about DonutChart.tsx ?